Library/Homebrew/os/mac/pkgconfig/11/expat.pc

```diff
@@ -1,4 +1,4 @@
-homebrew_sdkroot=/Library/Developer/CommandLineTools/SDKs/MacOSX11.0.sdk
+homebrew_sdkroot=/Library/Developer/CommandLineTools/SDKs/MacOSX11.sdk
```

Comment:
This doesn't exist.

Reply:
Yes, I've just found out and fixed it. I've asked for advice on how to best future-proof this, because the naming is somewhat inconsistent